src/engine/v3/V3Interpreter.v3

```diff
@@ -1128,14 +1128,15 @@ class V3Interpreter extends WasmStack {
 			I8X16_SWIZZLE => doSwizzle();
 			// ext:typed-continuation
 			CONT_NEW => {
-				var cont_index = codeptr.read_uleb32();
-				var cont_decl = ContDecl.!(frame.func.instance.heaptypes[cont_index]);
-				var func = Function.!(Value.Ref.!(pop()).val);
-				if (func == null) trap(TrapReason.NULL_DEREF);
+				// var cont_index = codeptr.read_uleb32();
```

Comment:
Can you keep the reading of the `cont_index` a separate line here? That way it's easier to see the decoding of immediates, rather than that being nested in an argument below.